Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:7.7.1-alpine
+
+RUN mkdir -p /usr/local/ui \
+&& apk update \
+&& apk upgrade \
+&& apk add -q vim \
+&& apk add -q nano;
+
+WORKDIR /usr/local/ui
+
+COPY package.json package.json
+COPY yarn.lock yarn.lock
+RUN yarn
+
+COPY . .
```

Comment:
Great! Maybe add a comment about this in `Dockerfile` so me and others won't ask this same question again in the future. ;-)